src/components/Person.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age && `I am ${age}`}</p>
+      {isMarried ? <p className="Person__partner">{`My ${partner}'s name is ${partnerName}`}</p> : 'I am not married'}
```

Comment:
You should render some string inside p tag
Now the last part of the ternary operator is rendered without any tag
Also will be better to compute married status string before return statement in your component
Such way can help to increase understanding your code and make it more clear
```suggestion
      <p ...>
        {isMarried
          ? {marriedStatus}
          : 'I am not married'
        }
      </p>
```